vendor/lualtek/lualtek-swsi-profile.yaml

```diff
@@ -0,0 +1,41 @@
+# Vendor profile ID, can be freely issued by the vendor
+# This vendor profile ID is also used on the QR code for LoRaWAN devices, see
+# https://lora-alliance.org/sites/default/files/2020-10/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf
+vendorProfileID: 1
+
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: '1.0.2'
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: 'RP001-1.0.2-RevB'
+
+# Whether the end device supports join (OTAA) or not (ABP)
+supportsJoin: true
+
+# Maximum EIRP
+maxEIRP: 16
+# Whether the end device supports 32-bit frame counters
+supports32bitFCnt: true
+
+# Whether the end device supports class B
+supportsClassB: false
+# If your device supports class B, uncomment the following fields:
+# Maximum delay for the end device to answer a MAC request or confirmed downlink frame (seconds)
+#classBTimeout: 60
+# Ping slot period (seconds)
+#pingSlotPeriod: 128
+# Ping slot data rate index
+#pingSlotDataRateIndex: 0
+# Ping slot frequency (MHz). Set to 0 if the band supports ping slot frequency hopping.
+#pingSlotFrequency: 869.525
+
+# Whether the end device supports class C
+supportsClassC: true
+# If your device supports class C, uncomment the following fields:
+# Maximum delay for the end device to answer a MAC request or confirmed downlink frame (seconds)
+#classCTimeout: 60
```

Comment:
if your device is class C you have to uncomment classCTimeout

Reply:
Thanks, done!